sources/core/Xenko.Core.Mathematics/Double2.cs

```diff
@@ -0,0 +1,1506 @@
+// Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
```

Comment:
Just a thought.
Should the copyright of new files still include "SiliconStudio"? I would assume their copyright does not extend past existing files. But I'm not a lawyer...

Reply:
Good question, but also, these probably count as existing files. I literally did a copy/paste Vector2/3/4 and find/replace "float" with "double" to create Double2/3/4. Then a few other minor things like fixing errors and converting between Vectors and Singles.